dataprofiler/labelers/data_processing.py

```diff
@@ -1556,7 +1556,8 @@ def __init__(
         elif isinstance(random_state, (list, tuple)) and len(random_state) == 3:
             # tuple required for random state to be set, lists do not work
             if isinstance(random_state[1], list):
-                random_state[1] = tuple(random_state[1])  # type: ignore
+                random_state = list(random_state)
```

Comment:
This might be a case for cast as opposed to a computation. I'm surprised `isinstance(random_state, (list, tuple))` isn't alleviating this.

Reply:
yeah just trick `mypy` with the cast rather than at execution time actually changing the type to `list`

Reply:
> This might be a case for cast as opposed to a computation. I'm surprised isinstance(random_state, (list, tuple)) isn't alleviating this.

It's because `random_state` could be a tuple, in which case indexed assignment isn't possible.

Is there any possibility that `random_state` is a tuple while `random_state[1]` is a list? If this will never happen we can safely cast here, or change the conditional to `if isinstance(random_state[1], list) and isinstance(random_state, list):`

Reply:
> Is there any possibility that random_state is a tuple while random_state[1] is a list?

Though, if this ever did happen in the past it would've thrown a runtime error.

Reply:
@taylorfturner Did this get resolved?

Reply:
> or change the line 1558 conditional to if isinstance(random_state[1], list) and isinstance(random_state, list):

I just committed this solution since it avoids casting